docs/datastore-emulator-setup.md

```diff
@@ -0,0 +1,27 @@
+# Datastore emulator setup
```

Comment:
Unlike third-party email service or recaptcha, the system cannot function without DB, so this should be part of the main development guide (i.e. no need for separate document).

Reply:
I see, thank you for correcting, so everything in this document should be placed inside `development.md` instead?

Reply:
I think it's good to create a new section just before `Testing` in `development.md`

Reply:
Alright, I'll make those changes.